public-api/src/main/java/org/pmiops/workbench/cdr/dao/ConceptService.java

```diff
@@ -145,16 +156,16 @@ public static String modifyMultipleMatchKeyword(String query, SearchType searchT
                         predicates.add(criteriaBuilder.greaterThan(matchExp, 0.0));
                     }
 
-                    if (standardConceptFilter.equals(StandardConceptFilter.STANDARD_CONCEPTS)) {
+                    if (standardConceptFilter.equals(StandardConceptFilterVal.STANDARD_CONCEPTS)) {
                             predicates.add(criteriaBuilder.or(standardConceptPredicates.toArray(new Predicate[0])));
-                    } else if (standardConceptFilter.equals(StandardConceptFilter.NON_STANDARD_CONCEPTS)) {
+                    } else if (standardConceptFilter.equals(StandardConceptFilterVal.NON_STANDARD_CONCEPTS)) {
                         predicates.add(
                                 criteriaBuilder.or(
                                         criteriaBuilder.or(criteriaBuilder.isNull(root.get("standardConcept"))),
                                         criteriaBuilder.and(nonStandardConceptPredicates.toArray(new Predicate[0]))
                                 ));
 
-                    } else if (standardConceptFilter.equals(StandardConceptFilter.STANDARD_OR_CODE_ID_MATCH)) {
+                    } else if (standardConceptFilter.equals(StandardConceptFilterVal.STANDARD_OR_CODE_ID_MATCH)) {
                         if (keyword != null) {
                             if(keyword != null){
```

Comment:
No need to wrap the same condition in itself
```suggestion
                        if (keyword != null) {
```

Reply:
Changed this.